test/e2e/ingress/resourcepushing.go

```diff
@@ -99,14 +98,56 @@ spec:
 		err = s.EnsureNumApisixUpstreamsCreated(1)
 		assert.Nil(ginkgo.GinkgoT(), err, "Checking number of upstreams")
 
+		s.NewAPISIXClient().GET("/ip").WithHeader("Host", "httpbin.com").Expect().Status(http.StatusOK)
+
+		// update
+		apisixRoute = fmt.Sprintf(`
+apiVersion: apisix.apache.org/v2alpha1
+kind: ApisixRoute
+metadata:
+  name: httpbin-route
+spec:
+  http:
+  - name: rule1
```

Comment:
should add cases : 
the diff `name` in the same `ApisixRoute`.
the same `name` in the diff `ApisixRoute`.

Reply:
Cannot capture your mean 100%.

Reply:
I mean add test cases:
1. In the same `ApisixRoute` yaml ,but change `spec.http.name` when update
2. In the different `ApisixRoute` yaml, but have the same `spec.http.name`, which means name is conflict.

Reply:
OK, will add it.